.github/workflows/linux.yml

```diff
@@ -384,6 +388,101 @@ jobs:
           path: ${{ env.INSTALL_TEST_DIR }}/TEST*.xml
           if-no-files-found: 'error'
 
+  Conformance_Tests:
+    needs: Build
+    defaults:
+      run:
+        shell: bash
+    runs-on: ubuntu-20.04-8-cores
+    strategy:
+      max-parallel: 2
+      fail-fast: false
+      matrix:
+        include:
+          # 'OP' for Opset, 'API' for API
+          - TEST_TYPE: 'OP'
```

Comment:
```suggestion
          - TEST_TYPE: 'Operations'
```

Reply:
I suggest keeping it as it is because the `run_conformance.py` script accepts `OP` and I do not want to make it uglier with something like `${{ matrix.TEST_TYPE == 'Operations' && 'OP' }}`.